### PR TITLE
Markdown lint conformance. Some spelling.

### PR DESCRIPTION
**Pull request by jhheider**
Nothing major. Biggest one is probably that markdownlint doesn't like multiple H1s in a document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,10 +46,10 @@ let route = sublimate { rq -> [String] in
 }
 ```
 
-> † Our find functions take optional IDs.\
```
Comment (mxcl) [inline README.md:49]: This backslash was deliberate, it forces a line-break

**Review by mxcl — CHANGES_REQUESTED** [review]
Man I need spell check in VSCode clearly. Must be an extension.

File: README.md (hunk 1 above)
Comment (jhheider) [reply README.md:49]: `shd101wyy.markdown-preview-enhanced` shows no difference with or with the line break (due to the &gt;s). Looks like that doesn't match github's implementation perfectly. Probably ought to report that.

**Review by mxcl — APPROVED** [review]

**Comment by jhheider** [comment]
> Man I need spell check in VSCode clearly. Must be an extension.

This PR brought to you by:
* streetsidesoftware.code-spell-checker
* davidanson.vscode-markdownlint

Code Spell Checker lets you set up both user and workspace dictionaries, which is great across projects.

**Pull request merged**